frontend/src/_styles/custom.scss

```diff
@@ -38,3 +38,8 @@ div[data-disabled='true'] {
 .accepted-files  .file-list {
   margin-bottom: -0.25rem;
 }
+
+//to adjust the placement of nav link menus id the dashboard
+.dropdown-menu{
```

Comment:
Instead of applying the styles globally, can we do this just for the nav link menu?